crates/language/src/target_language.rs

```diff
@@ -118,6 +118,91 @@ impl From<&TargetLanguage> for PatternLanguage {
     }
 }
 
+// see: https://github.com/clap-rs/clap/issues/4416
+impl ValueEnum for PatternLanguage {
+    // we need to implement the skip variants here ourselves now
+    fn value_variants<'a>() -> &'a [Self] {
+        &[
+            Self::Tsx,
+            Self::Html,
+            Self::Css,
+            Self::Json,
+            Self::Java,
+            Self::CSharp,
+            Self::Python,
+            Self::MarkdownBlock,
+            Self::Go,
+            Self::Rust,
+            Self::Ruby,
+            Self::Solidity,
+            Self::Hcl,
+            Self::Yaml,
+            Self::Sql,
+            Self::Vue,
+            Self::Toml,
+            Self::Php,
+            Self::PhpOnly,
+        ]
+    }
+
+    // https://github.com/getgrit/gritql/issues/445
+    // add any more language aliases here
+    // we need to implement the lowercase/rename transformations here ourselves now
+    fn to_possible_value<'a>(&self) -> Option<clap::builder::PossibleValue> {
```

Comment:
This isn't too bad, but I think we could probably avoid manually listing these. Instead it should be possible to use the existing `fmt` implementation to get the default string. Then add aliases by calling `get_file_extensions`.

Reply:
ahh yeah good point that is much cleaner and more concise 💪: e6f5da3

Reply:
Looks great, thanks!